plugins/dig-now.cpp

```diff
@@ -344,7 +344,11 @@ static void clean_ramp(MapExtras::MapCache &map, const DFCoord &pos) {
     if (is_wall(map, DFCoord(pos.x-1, pos.y, pos.z)) ||
             is_wall(map, DFCoord(pos.x+1, pos.y, pos.z)) ||
             is_wall(map, DFCoord(pos.x, pos.y-1, pos.z)) ||
-            is_wall(map, DFCoord(pos.x, pos.y+1, pos.z)))
+            is_wall(map, DFCoord(pos.x, pos.y+1, pos.z)) ||
+            is_wall(map, DFCoord(pos.x-1, pos.y-1, pos.z)) ||
+            is_wall(map, DFCoord(pos.x-1, pos.y+1, pos.z)) ||
+            is_wall(map, DFCoord(pos.x+1, pos.y-1, pos.z)) ||
+            is_wall(map, DFCoord(pos.x+1, pos.y+1, pos.z)))
```

Comment:
I don't think this is correct. This is checking for ramp usability, and diagonal walls do not make the ramp usable

Reply:
Maybe that was old behavior? DF wasn't marking them as unusable nor removing them when digging, so I included them. Lemme try creating with miners the normal way and test real quick if they get used by dwarves.

Reply:
Yeah, dwarves will use diagonal ramps without issue. That tracks to my experience in adventure mode using diagonal ramps for as long as I remember, so I'm not sure when this got changed, but evidently it did.